src/game/tick.c

```diff
@@ -113,6 +114,8 @@ static void advance_month(void)
     city_games_decrement_month_counts();
     city_gods_update_blessings();
     tutorial_on_month_tick();
+    scenario_events_progress_paused(1);
+    scenario_events_process_all();
```

Comment:
I was wondering how often you would be checking for the scenario events. A monthly check seems perfect 👍